bundles/core/pom.xml

```diff
@@ -153,26 +153,11 @@
                                                 <limit>
                                                     <counter>INSTRUCTION</counter>
```

Comment:
Should be `BRANCH` to get the branching coverage and not just the line coverage.